server/src/main/java/io/deephaven/server/resources/ServerResources.java

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright (c) 2016-2025 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.server.resources;
+
+import io.deephaven.configuration.Configuration;
+import io.deephaven.internal.log.LoggerFactory;
+import io.deephaven.io.logger.Logger;
+
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.ServiceLoader;
+import java.util.stream.Collectors;
+
+/**
+ * Helper to load resources based on configuration. Any resource specified in configuration be give a boolean "enabled"
+ * with true to allow the resources to be served and false to have them be excluded, with a default of true. Resources
+ * may also specify a positive integer "priority" to determine the order in which they are served, lowest first,
+ * allowing higher priority resources to override those with lower priority. Default priority is
```

Comment:
I think you mean "0" would beat out "10".  But when talking about lowest first it seems a bit confusing that higher beats out lower.